markets/spot-market/contracts/storage/Wrapper.sol

```diff
@@ -16,9 +16,9 @@ library Wrapper {
          */
         address collateralType;
         /**
-         * @dev sets if wrapping is enabled for given market id
+         * @dev amount of collateral that can be wrapped
          */
-        bool wrappingEnabled;
+        uint256 supplyCap;
```

Comment:
Unsure if this is right name for the variable. I think the market collateral module calls it "max depositable", Maybe this is like `maxWrappableAmount`?